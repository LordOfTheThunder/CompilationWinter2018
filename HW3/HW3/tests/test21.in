int potato(int a) {
	int t = a;
}
