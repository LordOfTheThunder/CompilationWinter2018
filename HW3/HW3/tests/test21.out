---end scope---
a INT -1
t INT 0
---end scope---
print (STRING)->VOID 1
printi (INT)->VOID 2
potato (INT)->INT 3
